lib/colonel_kurtz/renderer.ex

```diff
@@ -42,49 +47,48 @@ defmodule ColonelKurtz.Renderer do
     ]
   end
 
-  @spec block_view_module!(binary) :: module | none
-  def block_view_module!(type) do
-    case lookup_block_view_module!(type) do
+  @spec block_view_module(binary) :: module | none
+  def block_view_module(type) do
```

Comment:
This is now guaranteed to return a module. It also emits warnings instead of errors.